recipe/meta.yaml

```diff
@@ -31,6 +31,8 @@ requirements:
     - future
     - networkx
     - lxml
+  run_constrained:
+    - openforcefield ==0.8.0
```

Comment:
Maybe `<=0.8.0` is better here? Or do we need 0.8.0 only?

Reply:
That's probably better